app/controllers/comments_controller.rb

```diff
@@ -0,0 +1,20 @@
+class CommentsController < ApplicationController
+  def create
```

Comment:
- [ ] Please make sure that you have the new action in your comment controller and create a corresponding new.html.erb (which should contain your comment form) to be rendered on the `GET /posts/:post_id/comments/new(.:format) comments#new` route. :pray:

Reply:
Hi @Owusu-Desmond, I appreciate your suggestion, but I don't think it's necessary, the user can make a comment in the same post and it will be saved in the database. The way I see it, it's more practical to leave a comment in the same place instead of moving to another route.